pkg/inspect/inspect.go

```diff
@@ -227,3 +230,25 @@ type ImageResult struct {
 	Labels       map[string]string
 	Dangling     bool
 }
+
+// HealthCheckResults describes the results/logs from a healthcheck
+type HealthCheckResults struct {
+	// Status healthy or unhealthy
+	Status string `json:"Status"`
+	// FailingStreak is the number of consecutive failed healthchecks
+	FailingStreak int `json:"FailingStreak"`
+	// Log describes healthcheck attempts and results
+	Log []HealthCheckLog `json:"Log"`
+}
+
+// HealthCheckLog describes the results of a single healthcheck
+type HealthCheckLog struct {
+	// Start time as string
+	Start string `json:"Start"`
```

Comment:
I feel like these ought to be time.Time like StartedAt and FinishedAt

Reply:
im mimic'ing the actual healthcheck struct here. it gets converted along the way

Reply:
I'm a bit concerned about format... If these match the output format of StartedAt and FinishedAt when we print to terminal I'm fine with it